app/sidekiq/central_mail/submit_central_form686c_job.rb

```diff
@@ -14,15 +14,15 @@ class SubmitCentralForm686cJob
 
     sidekiq_options retry: false
 
-    attr_reader :claim, :form_686c_path, :form_674_path, :attachment_paths
+    attr_reader :claim, :form_path, :attachment_paths
 
     class CentralMailResponseError < StandardError; end
 
     def extract_uuid_from_central_mail_message(data)
       data.body[/(?<=\[).*?(?=\])/].split(': ').last if data.body.present?
     end
 
-    def perform(saved_claim_id, encrypted_vet_info, encrypted_user_struct)
+    def perform(saved_claim_id, encrypted_vet_info, encrypted_user_struct) # rubocop:disable Metrics/MethodLength
```

Comment:
I really like to discourage disabling these rubocop checks. I have never had a situation where I couldn't do some slight refactoring to just pull some stuff out into a function.

Reply:
split the method